vm/src/builtins/pystr.rs

```diff
@@ -294,44 +339,52 @@ impl PyStr {
     }
 
     pub fn as_str(&self) -> &str {
-        &self.value
```

Comment:
Safety comment? I'm haven't seen a specific rule throughout the project but I'm guessing comments describing safety should be required.

Also, wouldn't this benefit from being marked as `#[inlined(always)]?

Reply:
I added safey comments for every unsafe. About inline, I added one but I also tihnk these kind of tiny functions are easily marked as inline by compiler

Reply:
[Reading a bit about this](https://internals.rust-lang.org/t/inlining-policy-for-functions-in-std/14189/10), for private functions you're probably right. Cross crate public ones don't automatically get inlined. It's probably not harmful to add in this case.